src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml

```diff
@@ -416,6 +416,74 @@
 		</Setter>
 	</Style>
 
+	<Style x:Key="MaterialTextBoxEditableComboBoxStyle"
+		   TargetType="TextBox">
```

Comment:
You could probably use a BasedOn to the material textbox style and avoid needing to set things like the fontsize/weights/etc.